### PR TITLE
[search] remove from header bar

### PR DESCRIPTION
**Pull request by beets**
- Added a consolidated search page for CSE + place explorer search
- Remove from header bar
- Moved search related endpoints to a module and added tests

localhost_8080_s

### CODE REVIEW

**Comment by pradh** [comment]
[peanut gallery] should we include the good'ol BQ search too at the bottom (saying its a "legacy entity name search")?

**Comment by beets** [comment]
We can try it and monitor traffic / resource usage. @shifucun wdyt?

File: server/config/topic_page/poverty/USA.textproto [hunk 1]
```diff
@@ -18,6 +18,10 @@ metadata {
   topic_id: 'poverty'
   topic_name: 'Poverty'
   place_dcid: 'country/USA'
+  contained_place_types {
```
Comment (chejennifer) [inline server/config/topic_page/poverty/USA.textproto:21]: were these topic page changes supposed to be part of this pr?

**Review by chejennifer — APPROVED** [review]
nice!

File: server/config/topic_page/poverty/USA.textproto (hunk 1 above)
Comment (beets) [reply server/config/topic_page/poverty/USA.textproto:21]: it was a quick fix, but will be superseded by the more recent change #1441

**Pull request merged**